tests/test_e2e_15_maintenance.py

```diff
@@ -120,3 +120,38 @@ def test_010_create_mw_on_switch_should_move_evc(self):
         h11.cmd('ip link del vlan100')
         h3.cmd('ip link del vlan100')
 
+    def test_011_delete_mw_on_switch(self):
+        # 0. Start maintenance window
+        start = datetime.now() + timedelta(seconds=60)
+        end = start + timedelta(seconds=60)
+        payload = {
+            "start": start.strftime(TIME_FMT),
+            "end": end.strftime(TIME_FMT),
+            "items": [
+                "00:00:00:00:00:00:02"
+            ]
+        }
+
+        api_url = KYTOS_API + '/maintenance/'
+        # 1 Send get request to API to get maintenance schema
+        response = requests.get(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})["id"]
```

Comment:
Here you should make a POST request, not a GET, to create the maintenance window.
```suggestion
        response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})["id"]
```